packages/mobile/src/invite/saga.ts

```diff
@@ -105,14 +110,16 @@ export async function generateInviteLink(inviteCode: string) {
   return shortUrl
 }
 
-async function sendSms(toPhone: string, msg: string) {
+export async function sendSms(toPhone: string, msg: string) {
   return new Promise((resolve, reject) => {
     try {
       if (Platform.OS === 'android') {
+        // TODO: look into how to configure react-native-sms for Android so we can cut this package
```

Comment:
I tried to do this but got an ambiguous error. Someone who's more comfortable with android and the config settings should be able to do this pretty quickly.

Reply:
What error did you get? I thought maybe there was some issue here with not using the user's default messaging app, but I'm not 100% sure

Reply:
I didn't screen grab the exact error but I believe it had to do with the `SendSMSPackage.getInstance()` that you need to set in `getPackages` getting overwritten by some internal process. Couldn't find good documentation for solving and wasn't comfortable enough tinkering so I abandoned fairly quickly. Worth recreating the error message and creating an issue?